src/main/fc/fc_core.c

```diff
@@ -853,6 +853,9 @@ bool processRx(timeUs_t currentTimeUs)
         beeper(BEEPER_RC_SMOOTHING_INIT_FAIL);
     }
 #endif
+
+    pidSetAntiGravityState(IS_RC_MODE_ACTIVE(BOXANTIGRAVITY) || feature(FEATURE_ANTI_GRAVITY));
```

Comment:
Why was this moved here? Doing this in `rc_modes.c` / `fc_rc.c` like it was seems to be more appropriate.

Reply:
I'd have to ask @etracer65 :-)

Reply:
@ctzsnooze, @etracer65: I am still waiting for an answer to this.

Reply:
hopefully eTracer can explain why.  otherwise I'm happy to implement whatever suggestion solves the problem, it's over my head I'm afraid.

Reply:
Sorry, for some reason I wasn't receiving notifications on replies to this PR.

It's been a while so having to figure it out again, but if I recall the rationale was that the old anti-gravity was driven by the logic in fc_rc.c checking the feature/box state along with throttle movement and pushing a multiplier into the pid process using `pidSetItermAccelerator()`.  The new logic is completely in pid.c but also needs to know the active state (feature or box mode) but doesn't care about the multipler.  So the logic was normalized so that in fc_core.c (running at rx update rate) we push a flag into the PID controller that's used for both old and new modes.  This way the PID controller running at pid loop speed doesn't need to check the feature/box mode.  Also improves performance of the old mode since it was also previously checking the feature/box mode at pid loop speed in fc_rc.c `processRcCommand()`.

So in the big picture we're "pushing" the state into the PID controller but at RX update speed rather than "pulling" the state at PID loop speed.

Reply:
Thanks!  You're awesome, that was way over my head :-)